### PR TITLE
feat(kibbeh): redesign and implement new staff and contributor badges

### PR DESCRIPTION
**Pull request by ajmnz**
Implements the newly redesigned staff and (evolving) contributor badges.

Screenshot 2021-05-16 at 02 03 40

Screenshot 2021-05-16 at 02 05 26

### CODE REVIEW

**Comment by willdoescode** [comment]
These look great

**Review by goldyydev — APPROVED** [review]

**Review by amitojsingh366 — APPROVED** [review]
looks good to me!

File: kibbeh/src/modules/dashboard/ProfileBlockController.tsx [hunk 1]
```diff
@@ -33,18 +34,21 @@ export const ProfileBlockController: React.FC<ProfileBlockControllerProps> = ({}
...
     });
   }
+
   if (conn.user.contributions > 0) {
     badges.push({
-      content: "ƉC",
+      content: <ContributorBadge contributions={conn.user.contributions} />,
       variant: "primary",
       color: "white",
       title: t("components.userBadges.dhContributor"),
```
Comment (amitojsingh366) [inline kibbeh/src/modules/dashboard/ProfileBlockController.tsx]: although you should probably add the no. of contributions to the title, so ppl can see it on hover

Comment (amitojsingh366) [reply kibbeh/src/modules/dashboard/ProfileBlockController.tsx]:
```suggestion
      title: `${t("components.userBadges.dhContributor")} (${conn.user.contributions} ${t("pages.admin.contributions")})`,
```

File: kibbeh/src/ui/UserProfile.tsx [hunk 2]
```diff
@@ -19,27 +20,32 @@ export const UserProfile: React.FC<UserProfileProps> = ({
...
     });
   }
+
   if (user.contributions > 0) {
     badges.push({
-      content: "ƉC",
+      content: <ContributorBadge contributions={user.contributions} />,
       variant: "primary",
       color: "white",
       title: t("components.userBadges.dhContributor"),
```
Comment (amitojsingh366) [inline kibbeh/src/ui/UserProfile.tsx]:
```suggestion
      title: `${t("components.userBadges.dhContributor")} (${user.contributions} ${t("pages.admin.contributions")})`,
```

File: kibbeh/src/ui/VerticalUserInfo.tsx [hunk 3]
```diff
@@ -19,18 +20,20 @@ export const VerticalUserInfo: React.FC<VerticalUserInfoProps> = ({ user }) => {
...
+      naked: true,
     });
   }
   if (user.contributions > 0) {
     badges.push({
-      content: "ƉC",
+      content: <ContributorBadge contributions={user.contributions} />,
       variant: "primary",
       color: "white",
       title: t("components.userBadges.dhContributor"),
```
Comment (amitojsingh366) [inline kibbeh/src/ui/VerticalUserInfo.tsx]:
```suggestion
      title: `${t("components.userBadges.dhContributor")} (${user.contributions} ${t("pages.admin.contributions")})`,
```

**Comment by HoloPanio** [comment]
I really like the way these look!

**Pull request merged**